samples/sample_multi_transcode/src/transcode_utils.cpp

```diff
@@ -2979,6 +2979,34 @@ mfxStatus CmdProcessor::VerifyAndCorrectInputParams(TranscodingSample::sInputPar
     }
 #endif
 
+    mfxU16 mfxU16Limit = std::numeric_limits<mfxU16>::max();
+    if (InputParams.MaxKbps > mfxU16Limit || InputParams.nBitRate > mfxU16Limit ||
+        InputParams.InitialDelayInKB > mfxU16Limit || InputParams.BufferSizeInKB > mfxU16Limit)
+    {
+        mfxU32 maxVal = std::max<mfxU32>({ InputParams.MaxKbps,
+                                           InputParams.nBitRate,
+                                           InputParams.InitialDelayInKB,
+                                           InputParams.BufferSizeInKB });
+        InputParams.nBitRateMultiplier += (mfxU16)std::ceil(static_cast<double>(maxVal) / mfxU16Limit);
```

Comment:
I assume if user set TargetKbps > std::numeric_limits<mfxU16>::max() and some nBitRateMultiplier > 1 it's already a logical violation. Unzero nBitRateMultiplier means user is aware about TargetKbps  precision limitation so IMO better just to drop user defined value.

Reply:
removed